bitmovin/resources/enums/input_color_range.py

```diff
@@ -0,0 +1,11 @@
+import enum
+
+
+class InputColorRange(enum.Enum):
+    UNSPECIFIED = 'UNSPECIFIED'
+    MPEG = 'MPEG'
+    JPEG = 'JPEG'
+
+    @staticmethod
+    def default():
+        return InputColorRange.UNSPECIFIED
```

Comment:
do you actually need this default here? please get rid of that unspecified here, i don't think we need this. how does that look at the API side?

Reply:
This UNSPECIFIED is an value that is defined in the API spec